exp-player/addon/components/isp-radio-group/template.hbs

```diff
@@ -2,21 +2,37 @@
     {{#each options as |option index|}}
       <label class="text-center label-text {{if formatLabel formatLabel}}">
           {{#if labelTop}}
-            {{t option}}
+            {{#if hiddenOptions}}
+              {{#each hiddenOptions as |hidden|}}
+                <span class="{{if (eq hidden option) 'hidden'}}">{{t option}}</span>
+              {{/each}}
+            {{else}}
+                {{t option}}
+            {{/if}}
             {{radio-button value=(last-segment option) checked=value}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
           {{else}}
             {{#each-in labels as |ratings value|}}
               {{#if (eq value.rating option)}}
-                {{t value.label}}
+                <span class="{{if value.formatClass value.formatClass}}">
+                    {{t value.label}}
+                </span>
               {{/if}}
             {{/each-in}}
             {{radio-button value=(last-segment option) checked=value}}
-            {{t option}}
+            {{#if hiddenOptions}}
```

Comment:
The vertical alignment depends on the height, so if the option wraps onto a second line the radio button gets pushed up and if there is no option label the radio button gets pushed down. It's not an issue for this particular question since the number options will not be translated. But it could definitely be an issue for questions where the options are words and not numbers.